### DISCUSSION TITLE
Custom Button add to hotkeys

### DISCUSSION THREAD

**Discussion by Jassi-X** [arkypita/LaserGRBL#1862] [Ideas]
Good day, anyone know how to add one of the custom Buttons to the hotkeys list or how to access it by only pressing a button on the keyboard. Trying to run the Blink button from a wireless keyboard / pendant.

Any help would be greatly apricated.
![Blink Button Hotkeys](https://user-images.githubusercontent.com/109039704/178157973-40184b83-f2de-484e-8851-7aa8ed83f074.png)
![Blink Button](https://user-images.githubusercontent.com/109039704/178157974-2f5e7d6e-6175-4afb-b1b2-f0deb213116c.png)

**Comment by arkypita** [comment]
Scroll down

![image](https://user-images.githubusercontent.com/8782035/178160453-3b4bfa67-94a3-4c6c-bef5-de1b4c14e733.png)

**Reply by Jassi-X** [reply]
Thank you very much. Sorted

Do you know of any plug ins that support IP Camera's?
